R/utilities-units.R

```diff
@@ -95,38 +95,22 @@ getBaseUnit <- function(dimension) {
 #' @export
 toBaseUnit <- function(quantityOrDimension, values, unit, molWeight = NULL, molWeightUnit = NULL) {
```

Comment:
Lot's of code duplication in `toBaseUnit()` - I have changed it to using `toUnit()` (which now also exists earlier if the target unit equals the source unit)